dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceApplication.java

```diff
@@ -213,7 +215,7 @@ public class DockstoreWebserviceApplication extends Application<DockstoreWebserv
     public static final String SLIM_WORKFLOW_FILTER = "slimWorkflowFilter";
     public static final String SLIM_VERSION_FILTER = "slimVersionFilter";
 
-    public static final String SLIM_USER_FILTER = "slimUserFilter";
```

Comment:
Got rid of this because the `SLIM_USER_FILTER` was excluding properties that already have `@JsonIgnore`.